test/performance/io/format_sam_benchmark.cpp

```diff
@@ -0,0 +1,225 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+// !\author Clemens Cords (clemens.cords@fu-berlin.de)
```

Comment:
```suggestion
//!\author Clemens Cords (clemens.cords@fu-berlin.de)
```

The only non header file with author information 🤔

Reply:
I know.. but I felt bad removing it..